packages/desktop-ui-lib/src/lib/logger/logger.component.html

```
@@ -0,0 +1,77 @@
+			<nb-card-body>
+				<div class="log-viewport" #scrollContainer (scroll)="onScroll($event)">
+
+					@if (isLoadingMore()) {
+					<div class="load-more-bar">
+						<div [nbSpinner]="true" nbSpinnerSize="tiny" nbSpinnerStatus="primary"
+							class="load-more-spinner"></div>
+						<span>Loading older entries…</span>
+					</div>
+					}
+
+					@if (!hasMore() && filteredLogs().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)